recipe/meta.yaml

```
@@ -17,7 +17,7 @@ build:
 requirements:
     build:
         - {{ compiler('c') }}
-        - {{ stdlib("c") }}
+        - libcxx     # [osx]
         - make       # [unix]
     host:
         - zlib
```

```suggestion
        - zlib {{ zlib }}
```
because it's in cbc.yaml https://github.com/AnacondaRecipes/aggregate/blob/master/conda_build_config.yaml#L240